### ISSUE TITLE
Facebook not pulling pictures + Lost Google Review Rich Snippets

### ISSUE THREAD

**Issue by Unpleasant** [gnotaras/wordpress-add-meta-tags#5]
Hey! Was wondering if anyone else had this problem, I just noticed that Facebook is no longer pulling up pictures from my articles when pasting them, and in addition my Rich Snippets for reviews on Google search results are also not displaying anymore. (The star ratings displayed on Google's search results when you search for a particular service or product review.)

Both of these were working completely fine a few weeks ago, and I haven't updated anything since then. 

Anyone else have this issue/can offer advice?

My website is http://www.eliteenvy.com - The Google rich snippet schema debug tool shows everything as working (but the rich snippets don't display) - The Facebook debugger does show an issue with my image display:

og:image was not defined, could not be downloaded or was not big enough. Please define a chosen image using the og:image metatag, and use an image that's at least 200x200px and is accessible from Facebook. Image 'http://www.eliteenvy.com/wp-content/uploads/2014/01/Sleeping-Kitty-1021x580.jpg' will be used instead. Consult http://developers.facebook.com/docs/sharing/webmasters/crawler for more troubleshooting tips.

Basically Facebook tries to grab the original filename of the picture first, and after it fails, it instead grabs a different one that is working. 

The weird part is that I haven't made any changes for weeks, and suddenly they quit working on me.

**Comment by gnotaras** [comment]
Hi,

> I just noticed that Facebook is no longer pulling up pictures from my articles when pasting them

There is a problem with the images. The image links that are generated by the plugin point to non existing images. Have you made any changes regarding the original images? The 'full' image size is what add-meta-tags uses by default in the metadata (can be customized). But that image url returns a 404 Not Found error when checked on your web site. The absence of the images is not related to the plugin. The plugin uses the WordPress API to get the image URLs. They should be there.

In order to use another image size in the metadata, please [check this section](http://www.codetrax.org/projects/wp-add-meta-tags/wiki/Plugin_Functionality_Customization#Custom-Default-Image-Sizes).

Also, use the opengraph debug tool to test the pages.

> my Rich Snippets for reviews on Google search results are also not displaying anymore. (The star ratings displayed on Google's search results when you search for a particular service or product review.)

How were the rating being added to the page? I don't see any rating related object in the output of the Google Structured Data Tool. Can you point me to a specific page?

Also, there are multiple _Article_ schema.org objects.

> The weird part is that I haven't made any changes for weeks, and suddenly they quit working on me.

Can you confirm that the images exist in the filesystem using an ftp client? For example, the following image should exist: `/wp-content/uploads/2014/12/6397510407_e31b247711_o-e1420440767892.jpg`

When trying to access it over the web, your web site returns a 404 error.

**Comment by gnotaras** [comment]
I looked into this again and I'm afraid I'll have to mark the issue as invalid. There is indeed a problem with your web site, but this problem is not related to the plugin, since the metadata is generated as expected.

However, I'll keep this issue open for a little while in case you have feedback related to Add-Meta-Tags.

Kind Regards,
George

**Comment by Unpleasant** [comment]
Hey, thanks for the quick reply! I didn't think it was the plugin cause I've never had any issues with it before, (and I could tell the image was just pulling a 404ed image location) but then I also had the problem with the Rich snippets and since they both rely on this plugin to work (schema and og tags) I thought I would ask just incase. 

Thanks a ton for the information! 

(The rich snippet I was talking about: https://www.google.com/?gws_rd=ssl#q=manageflitter+review 
As you can see the 5th ranked g2crowd is displaying a rating in the search results, although mine, which is ranked 2nd, isn't showing it anymore. It was a few weeks ago, and now when I check they no longer appear. The debug tool tells me everything is fine on that though, which boggles my mind even worse, i rehighlighted everything and resubmitted via Google webmaster, so maybe I'll get lucky and it will work itself out.)

Thanks again for the info regarding the og image, I'll have to figure out what changed over the past few weeks :)

**Comment by gnotaras** [comment]
Oh I see. I was looking for a Review schema.org entity in the metadata.

Your feedback is always welcome and much appreciated! :-)

**Comment by gnotaras** [comment]
Hi, you seem to be working on the issue on your web site. I'm going to close this ticket now. Please, feel free to open a new ticket, if you encounter any problems with Add-Meta-Tags.

Kind Regards,
George
